yt/visualization/fixed_resolution.py

```diff
@@ -771,9 +771,9 @@ def __getitem__(self, item):
         # requested
         info = self._get_info(item)
         if density:
-            width = self.data_source.width
-            norm = width[self.xax] * width[self.yax] / np.prod(self.buff_size)
-            norm = norm.in_base()
+            dpx = (self.bounds[1] - self.bounds[0]) / self.buff_size[0]
```

Comment:
Are these guaranteed to be in code length?  Or will they carry units?

Reply:
@matthewturk now you have the good catch--see https://github.com/yt-project/yt/pull/4231/commits/867db30737dc04a40ceb0cfeed66ddf222969904